<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientFileWindow</class>
 <widget class="QMainWindow" name="PatientFileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Картотека пациентов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="surname_for_filter">
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Фамилия</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="name_for_filter">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Имя</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="patronymic_for_filter">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Отчество</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reset_filter_note">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="table_patient_db"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="surname_label">
            <property name="text">
             <string>Фамилия:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="surname"/>
          </item>
          <item>
           <widget class="QLabel" name="name_label">
            <property name="text">
             <string>Имя:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="name"/>
          </item>
          <item>
           <widget class="QLabel" name="patronymic_label">
            <property name="text">
             <string>Отчество:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="patronymic"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="add_note">
              <property name="text">
               <string>Добавить</string>
              </property>
              <property name="shortcut">
               <string>Return</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="change_note">
              <property name="text">
               <string>Изменить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="delete_note">
              <property name="text">
               <string>Удалить</string>
              </property>
              <property name="shortcut">
               <string>Del</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="export_cvs"/>
    <addaction name="import_cvs"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="export_cvs">
   <property name="text">
    <string>Экспорт в cvs</string>
   </property>
  </action>
  <action name="import_cvs">
   <property name="text">
    <string>Импорт из cvs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
